ext/lvgl/lvgl.lb

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Raphael Lehmann
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+def init(module):
+    module.name = ":lvgl"
+    module.description = """
+# LittlevGL
+
+LittlevGL is a free and open-source graphics library providing everything you need
+to create embedded GUI with easy-to-use graphical elements, beautiful visual effects
+and low memory footprint.
+
+- https://littlevgl.com/
+- https://github.com/littlevgl/lvgl
+"""
+
+def prepare(module, options):
+
+    module.add_option(
+        NumericOption(
+            name="resolution_horizontal_max",
+            description="Maximal horizontal resolution to support by lvgl",
+            minimum=1,
+            default=480))
+    module.add_option(
+        NumericOption(
+            name="resolution_vertical_max",
+            description="Maximal vertical resolution to support by lvgl",
+            minimum=1,
+            default=320))
+    module.add_option(
+        EnumerationOption(
+            name="color_depth",
+            description="Color depth",
+            enumeration={
+                "OneBitPerPixel": 1,
+                "RGB233": 8,
+                "RGB565": 16,
+                "ARGB8888": 32,
+                },
+            default="RGB565"))
+    module.add_option(
+        NumericOption(
+            name="dpi",
+            description="Dot Per Inch: used to initialize default sizes",
+            minimum=1,
+            default=100))
+    module.add_option(
+        BooleanOption(
+            name="logging",
+            description="Enable LVGL logging",
+            default=False))
+    module.add_option(
+        EnumerationOption(
+            name="log_level",
+            description="Verbosity of the log",
+            enumeration={
+                "Debug": "LV_LOG_LEVEL_TRACE",
+                "Info": "LV_LOG_LEVEL_INFO",
+                "Warning": "LV_LOG_LEVEL_WARN",
+                "Error": "LV_LOG_LEVEL_ERROR",
+                },
+            default="Warning"))
```

Comment:
Do we want to collapse the `logging` option into the `log_level` option with a "Disabled" entry? It would require the user to choose the log_level (ie. no default log level), but it would be one less option.